string object_type
geometry_msgs/Pose object_pose
---
geometry_msgs/Pose pose_pickup
geometry_msgs/Pose pose_approach
float32 open_width
